### ISSUE TITLE
Async Google AdSense Code inside lazy-ads?

### ISSUE THREAD

**Issue by baszero** [madgex/lazy-ads#15]
Anyone having experience whether it makes sense to wrap the async Google AdSense snippets inside the lazy-ads? I would say yes because google async only means it runs in parallel to the page load, inside the lazy-ads wrapper it would load after page load, correct?

**Comment by Niresh12495** [comment]
Yes as because domready is used this will increase user experience positively
